_includes/pbc/all/install-features/202212.0/install-the-customer-account-management-feature.md

```diff
@@ -0,0 +1,1029 @@
+| CustomerCriteriaFilterTransfer             | class | created | src/Generated/Shared/Transfer/CustomerCriteriaFilterTransfer.php             |
+| CustomerIdentifierTransfer                 | class | created | src/Generated/Shared/Transfer/CustomerIdentifierTransfer.php                 |
+| CustomerResponseTransfer                   | class | created | src/Generated/Shared/Transfer/CustomerResponseTransfer.php                   |
+| JwtTokenTransfer                           | class | created | src/Generated/Shared/Transfer/JwtTokenTransfer.php                           |
+| OauthAccessTokenDataTransfer               | class | created | src/Generated/Shared/Transfer/OauthAccessTokenDataTransfer.php               |
+| OauthAccessTokenValidationRequestTransfer  | class | created | src/Generated/Shared/Transfer/OauthAccessTokenValidationRequestTransfer.php  |
+| OauthAccessTokenValidationResponseTransfer | class | created | src/Generated/Shared/Transfer/OauthAccessTokenValidationResponseTransfer.php |
+| OauthClientTransfer                        | class | created | src/Generated/Shared/Transfer/OauthClientTransfer.php                        |
+| OauthErrorTransfer                         | class | created | src/Generated/Shared/Transfer/OauthErrorTransfer.php                         |
+| OauthGrantTypeConfigurationTransfer        | class | created | src/Generated/Shared/Transfer/OauthGrantTypeConfigurationTransfer.php        |
+| OauthRefreshTokenCollectionTransfer        | class | created | src/Generated/Shared/Transfer/OauthRefreshTokenCollectionTransfer.php        |
+| OauthRefreshTokenTransfer                  | class | created | src/Generated/Shared/Transfer/OauthRefreshTokenTransfer.php                  |
+| OauthRequestTransfer                       | class | created | src/Generated/Shared/Transfer/OauthRequestTransfer.php                       |
+| OauthResponseTransfer                      | class | created | src/Generated/Shared/Transfer/OauthResponseTransfer.php                      |
+| OauthScopeRequestTransfer                  | class | created | src/Generated/Shared/Transfer/OauthScopeRequestTransfer.php                  |
+| OauthScopeTransfer                         | class | created | src/Generated/Shared/Transfer/OauthScopeTransfer.php                         |
+| OauthTokenCriteriaFilterTransfer           | class | created | src/Generated/Shared/Transfer/OauthTokenCriteriaFilterTransfer.php           |
+| OauthUserTransfer                          | class | created | src/Generated/Shared/Transfer/OauthUserTransfer.php                          |
+| RevokeRefreshTokenRequestTransfer          | class | created | src/Generated/Shared/Transfer/RevokeRefreshTokenRequestTransfer.php          |
+| RevokeRefreshTokenResponseTransfer         | class | created | src/Generated/Shared/Transfer/RevokeRefreshTokenResponseTransfer.php         |
```

Comment:
I assume `SessionEntityRequest` and `SessionEntityResponse` will be generated since the feature contains respective session validation modules

Reply:
Yes, correct. We have a separate section `Install feature frontend` below where these transfers are mentioned (the similar approach as for the Agent Assist feature).